### DISCUSSION TITLE
Actions on page load

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#3876] [Ideas]
Is there a way to run actions on page load similar to the button actions?
Basically, I want to check and set a few App State values on page load.

Something like:

```js
$(function() {
    // i want to be able to trigger actions similar to the button actions
});
```

Many thanks!

**Comment by mjashanks** [comment]
not at the moment, but we should add this!

**Reply by Shadowfita** [reply]
Definitely add this please!

**Reply by ryzam** [reply]
Latest version of Budibase has action on screen load

![image](https://user-images.githubusercontent.com/597759/185731947-80525964-9fcd-4569-a739-86bac805a99d.png)
